gridstatus/pjm.py

```diff
@@ -941,6 +941,58 @@ def extract_short_name(row):
 
         return data
 
+    @support_date_range(frequency=None)
+    def get_it_sced_lmp_5_min(self, date, end=None, verbose=False):
+        """Get 5 minute LMPs from the Integrated Forward Market (IFM)"""
+
+        if date == "latest":
+            return self.get_it_sced_lmp_5_min("today", verbose=verbose)
+
+        params = {
+            "fields": (
+                "case_approval_datetime_utc,datetime_beginning_utc,itsced_lmp,marginal_congestion,marginal_loss,pnode_id,pnode_name"  # noqa: E501
+            ),
+        }
+
+        data = self._get_pjm_json(
+            "five_min_itsced_lmps",
+            start=date,
+            end=end,
+            params=params,
+            verbose=verbose,
+            interval_duration_min=5,
+        )
+
+        # TODO: are there other columns we want to rename?
+        data = data.rename(
+            columns={
+                "case_approval_datetime_utc": "Publish Time",
```

Comment:
im wondering if we want to keep this the name as it is published, since it feels like the column is descriptive of something specific. 

Im having a hard time understanding what this is exactly and it's the kind of thing I'd like to ask connor...

to unblock thing I'm going to vote we just name it `Case Approval Time`.

Reply:
Changed to `"Case Approval Time"`.